ebu_tt_live/xsd1.1/ebutt_datatypes.xsd

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016, EBU, www.ebu.ch -->
+<!-- Version of XML Schema: 0.9 -->
+<!-- Creation: 05/04/2016 -->
+<!--  The publication of the EBU-TT XML Schema for EBU-Part 3 is intended to support the implementation of the specification in EBU-Tech 3370 version 0.9. 
+Please note that the EBU-TT XML Schema is a helping document and NOT normative but informative.-->
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:ebuttdt="urn:ebu:tt:datatypes"
+	targetNamespace="urn:ebu:tt:datatypes">
+	<xs:simpleType name="cellResolutionType">
+		<xs:restriction base="xs:token">
+			<xs:pattern value="[1-9][0-9]*\s[1-9][0-9]*"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="frameRateMultiplierType">
+		<xs:restriction base="xs:token">
+			<xs:pattern value="[1-9][0-9]*\s[1-9][0-9]*"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="extentType">
+		<xs:restriction base="xs:token">
+			<xs:pattern value="([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="fontFamilyType">
+		<xs:restriction base="xs:string"/>
+	</xs:simpleType>
+	<xs:simpleType name="fontSizeType">
+		<xs:restriction base="xs:token">
+			<xs:pattern
+				value="([+-]?\d*\.?\d+(px|c|%))|([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))"
+			/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="lineHeightType">
+		<xs:union memberTypes="ebuttdt:lengthType">
+			<xs:simpleType>
+				<xs:restriction base="xs:token">
+					<xs:enumeration value="normal"/>
+				</xs:restriction>
+			</xs:simpleType>
+		</xs:union>
+	</xs:simpleType>
+	<xs:simpleType name="colorType">
+		<xs:union
+			memberTypes="ebuttdt:rgbHexColorType ebuttdt:rgbaHexColorType ebuttdt:rgbColorType ebuttdt:rgbaColorType ebuttdt:namedColorType"
+		/>
+	</xs:simpleType>
+	<xs:simpleType name="originType">
+		<xs:restriction base="xs:token">
+			<xs:pattern value="([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))"/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="paddingType">
+		<xs:restriction base="xs:token">
+			<xs:pattern
+				value="([+-]?\d*\.?\d+(px|c|%))|([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))|([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))|([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))"
+			/>
+		</xs:restriction>
+	</xs:simpleType>
+	<xs:simpleType name="timingType">
+		<xs:union
+			memberTypes="ebuttdt:smpteTimingType ebuttdt:mediaTimingType ebuttdt:clockTimingType"/>
+	</xs:simpleType>
+	<xs:simpleType name="durationTimingType">
+		<xs:union 
+			memberTypes="ebuttdt:clockTimingType ebuttdt:mediaTimingType"/>
+	</xs:simpleType>
+	<xs:simpleType name="smpteTimingType">
+		<xs:restriction base="xs:string">
+			<xs:pattern value="[0-2][0-9]:[0-5][0-9]:[0-5][0-9]:[0-9][0-9]"/>
```

Comment:
Thanks @frans-ebu - the spec reference is actually to the 2008 version, so it would be good to know if they're different, but in the meantime we should take the 2016 version and also update the spec reference to point to it.